[2024-09-04T19:19:36.578+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-04T19:19:36.592+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_input_2sum.get_sum manual__2024-09-04T19:19:21+00:00 [queued]>
[2024-09-04T19:19:36.606+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_input_2sum.get_sum manual__2024-09-04T19:19:21+00:00 [queued]>
[2024-09-04T19:19:36.609+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-09-04T19:19:36.621+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): get_sum> on 2024-09-04 19:19:21+00:00
[2024-09-04T19:19:36.631+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1114) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-04T19:19:36.633+0000] {standard_task_runner.py:72} INFO - Started process 1115 to run task
[2024-09-04T19:19:36.631+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'user_input_2sum', 'get_sum', 'manual__2024-09-04T19:19:21+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/python_operator_dag_with_userinput.py', '--cfg-path', '/tmp/tmpfjbce6y_']
[2024-09-04T19:19:36.635+0000] {standard_task_runner.py:105} INFO - Job 119: Subtask get_sum
[2024-09-04T19:19:36.694+0000] {task_command.py:467} INFO - Running <TaskInstance: user_input_2sum.get_sum manual__2024-09-04T19:19:21+00:00 [running]> on host 626f2e28dec2
[2024-09-04T19:19:36.745+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abhay' AIRFLOW_CTX_DAG_ID='user_input_2sum' AIRFLOW_CTX_TASK_ID='get_sum' AIRFLOW_CTX_EXECUTION_DATE='2024-09-04T19:19:21+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-04T19:19:21+00:00'
[2024-09-04T19:19:36.747+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-04T19:19:36.759+0000] {logging_mixin.py:190} INFO - Received num1: 10, num2: 24
[2024-09-04T19:19:36.761+0000] {logging_mixin.py:190} INFO - Sum: 1024
[2024-09-04T19:19:36.762+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-04T19:19:36.769+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-04T19:19:36.770+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=user_input_2sum, task_id=get_sum, run_id=manual__2024-09-04T19:19:21+00:00, execution_date=20240904T191921, start_date=20240904T191936, end_date=20240904T191936
[2024-09-04T19:19:36.813+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-04T19:19:36.824+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-04T19:19:36.826+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
